### PR TITLE
infra/gcp: Grant Release Manager Admins roles/cloudkms.cryptoOperator

### PR DESCRIPTION
**Pull request by justaugustus**
tl;dr -- I started playing around w/ [`cosign`](https://github.com/sigstore/cosign) and [need more permissions](https://github.com/sigstore/cosign/blob/main/KMS.md#gcp) 😅 

While Release Manager Admins have
roles/cloudkms.cryptoKeyEncrypterDecrypter, this role is not
sufficient to test signing/verification operations e.g., `cosign`

roles/cloudkms.cryptoOperator grants encrypt, decrypt, sign, and
verify access.

Before, with roles/cloudkms.cryptoKeyEncrypterDecrypter:
- cloudkms.cryptoKeyVersions.useToDecrypt
- cloudkms.cryptoKeyVersions.useToEncrypt
- cloudkms.locations.get
- cloudkms.locations.list
- resourcemanager.projects.get

After, with roles/cloudkms.cryptoOperator:
- cloudkms.cryptoKeyVersions.useToDecrypt
- cloudkms.cryptoKeyVersions.useToEncrypt
- cloudkms.cryptoKeyVersions.useToSign
- cloudkms.cryptoKeyVersions.useToVerify
- cloudkms.cryptoKeyVersions.viewPublicKey
- cloudkms.locations.*
- resourcemanager.projects.get

Signed-off-by: Stephen Augustus <foo@auggie.dev>

/hold for actuation
cc: @kubernetes/release-engineering @kubernetes/sig-release-leads

### CODE REVIEW

**Review by cpanato — APPROVED** [review]
/lgtm

/hold for @spiffxp / @ameukam

**Review by saschagrunert — APPROVED** [review]

**Review by xmudrii — APPROVED** [review]

File: infra/gcp/lib.sh [hunk 1]
```diff
@@ -313,7 +313,7 @@ function empower_group_for_kms() {
 
     local roles=(
         roles/cloudkms.admin
-        roles/cloudkms.cryptoKeyEncrypterDecrypter
+        roles/cloudkms.cryptoOperator
```
Comment (ameukam) [inline infra/gcp/lib.sh:316]:
```suggestion
		roles/cloudkms.cryptoKeyEncrypterDecrypter
        roles/cloudkms.signerVerifier
```

Comment (justaugustus) [reply infra/gcp/lib.sh:316]: @ameukam -- I considered your role suggestion, but opted for Operator, since it's just about equivalent and consolidates a line. Is there a strong preference towards signerVerifier?

Comment (ameukam) [reply infra/gcp/lib.sh:316]: @justaugustus I think it's better to drop the `cloudkms.locations.*` for something less permissive. I can open a followup PR if it's not enough.
